README.md

```diff
@@ -1,11 +1,11 @@
 # @embroider/app-blueprint
 
-**Very** experimental blueprint for scaffolding ember v2 apps with Vite
+**Very** experimental blueprint for scaffolding Ember v2 apps with Vite
 
 This is likely to change on a daily basis so you have to keep up to date with the changes to expect it work. Use [ember-cli-update](https://github.com/ember-cli/ember-cli-update) to update to the latest version.
 
 ## Usage
 
 ```bash
-ember new my-fancy-app -b @embroider/app-blueprint --pnpm
-```
\ No newline at end of file
+npx ember-cli@latest new my-fancy-app -b @embroider/app-blueprint --pnpm
```

Comment:
Alternatively, we could also document the pnpm specific command:

```
pnpm dlx ember-cli@latest new my-fancy-app -b @embroider/app-blueprint --pnpm
```

Reply:
firstly, you can just `pnpx` and it works ;) 

but anyone who knows pnpm probably also knows how to convert from npx commands already so I'm happy to leave npx here 👍